app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -261,6 +266,20 @@ class ResourcesFragment(val root: Path?, val path: Path?): MvpAppCompatFragment(
         dialog = alertBuilder.show()
     }
 
+    private fun sortingCategorySelected(itemID: Int): Sorting{
+        return when(itemID) {
+            R.id.rb_default -> {
+                Sorting.DEFAULT
+                throw AssertionError("As-is sorting is initial, unsorted order")
```

Comment:
This is interesting case. Sometimes, a user might want "as-is" sorting, because it is slightly faster and in some cases order doesn't matter. E.g. later we will have "shuffle" button, so we don't need to sort elements if the user wants to use it. But if we change sorting and remember it, then we can't switch to "as-is". So, we should make it selectable, but since it doesn't make sense to "unsort" resources it should be active only starting next time.